docs/sensors/smartspot.md

```diff
@@ -1,11 +1,199 @@
 
-## How to add Air Quality sensors
+To achieve this, we take advantage from technologies like Physical Web. Users interact via their smartphones thanks to our device that sends &quot;push&quot; notifications with digital content through Bluetooth without the need to install native Apps.
 
-## Register your SmartSpot in the IoT Agent
+In order to monitorize the air pollution, Smart Spot measures the NO2, CO, SO2 and O3 at specific points in real time. We have a sophisticated high precision lab, with a Mass Flow Controller and a Zero Air Generator, where through Machine Learning algorithms we improve the precision of our sensor measures reducing the effect of cross sensitivity.
 
-## Register your Air Quality Dashboard in the SmartSpot
+For the care and management of data collected by the Smart Spot a maintenance platform is needed, our device can be managed remotely by platforms which use OMA LWM2M as FIWARE. We deploy FIWARE to manage Smart Spot data and also, we integrate our solution in already existing FIWARE ecosystems.
+
+Optionally, this system provides information about crowds in specific areas, detecting people with smartphone WiFi switched-on.
+
+
+## **What do you need?**
+
+### What is included with The Smart Spot Starter Kit:
+
+1. ESP32 DevKitC.
+
+![ESP32 DevKitc](./images/ESP32-DevKitC.png)
+
+2. Starter Kit firmware
+
+ 	*  Physical Web
+	* Crowd Monitoring
+	* LwM2M server
+	* Integration with I2C sensors
+	* GPIO driver
+
+3. Expansion Board:
+ 	
+ 	* Temperature, Humidity and Pressure Sensor.
+	* Accelerometer and Gyroscope.
+	* Luminosity Sensor
+	* RGB led
+	* GPIO Led
+
+4. Micro USB Cable
+5. Smart Spot Air Quality Expansion Board
+6. I2C Cable
+ 
+![I2C Cable](./images/I2C.png)
+
+7. 4 x Gas Sensors:
+ 	* Sulfur Dioxide (Red).
+ 	* Ozone + Nitrogen Dioxide (Yellow).
+ 	* Carbon Monoxide (Green).
+ 	* Nitrogen Dioxide (Orange).
+
+![Gas Sensor](./images/sensor.png)
+
+You will need a PC loaded with Windows, Linux or Mac OS with internet connection in order to download the Starter Kit firmware and the ESP32 toolchain and ESP-IDF.
+
+## **Getting Started**
+
+This chapter is a guide for Windows users. In case you use another OS, you can find further information in the following link
+
+[http://esp-idf.readthedocs.io/en/latest/get-started/index.html](http://esp-idf.readthedocs.io/en/latest/get-started/index.html)
+
+
+1) First, you need to download the ESP32 toolchain by clicking in this link: [https://dl.espressif.com/dl/esp32\_win32\_msys2\_environment\_and\_toolchain-20180110.zip](https://dl.espressif.com/dl/esp32_win32_msys2_environment_and_toolchain-20180110.zip). Unzip it and place somewhere safe. This guide assumes the environment is placed in _C:\ _ .
+
+2) You will also need the ESP32 API. In order to download it, open a terminal, navigate to a directory where you want to place the IDF and use these commands:
+
+~~~
```

Comment:
use canonical md code and indent it for proper alignment below the list

1. xxxx

    ```bash
    $ git clone --recursive https://github.com/espressif/esp-idf.git
 
    $ git checkout tags/v3.0-rc1
 
    $ git submodule update –init
    ```